resharper/resharper-unity/src/CSharp/Daemon/Stages/Analysis/UnityEventFunctionAnalyzer.cs

```diff
@@ -23,9 +23,12 @@ namespace JetBrains.ReSharper.Plugins.Unity.CSharp.Daemon.Stages.Analysis
         })]
```

Comment:
I think we need to add `UnityCodeInsightsHighlighting` into this list of types, here and in the other analysers.